.github/workflows/os-compatibility-test.yaml

```diff
@@ -15,9 +15,31 @@ on:
       - '**/*.md*'
```

Comment:
Did you intend to remove `**/*.md*` from the ignore list?

Reply:
Yes, I did. Matching works a bit different with the paths-ignore job and the built-in `paths-ignore` behavior.